pkg/tenant/config/config.go

```diff
@@ -62,14 +62,15 @@ type QuarterdeckConfig struct {
 
 // Configures an SDK connection to Ensign for pub/sub.
 type SDKConfig struct {
-	Enabled          bool   `default:"true" yaml:"enabled"`
-	TopicName        string `split_words:"true" default:"ensign.metatopic.topics"`
-	ClientID         string `split_words:"true"`
-	ClientSecret     string `split_words:"true"`
-	Endpoint         string `default:"ensign.rotational.app:443"`
-	AuthURL          string `split_words:"true" default:"https://auth.rotational.app"`
-	Insecure         bool   `default:"false"`
-	NoAuthentication bool   `split_words:"true" default:"false"`
+	Enabled          bool          `default:"true" yaml:"enabled"`
+	TopicName        string        `split_words:"true" default:"ensign.metatopic.topics"`
```

Comment:
Remind me, does the Go SDK support topic names for subscribe?

Reply:
It does - or at least it should; I think there may be a minor bug there -- but I'll fix it if you come across it.